console/src/app/app-routing.module.ts

```diff
@@ -10,15 +9,15 @@ import { OrgCreateComponent } from './pages/org-create/org-create.component';
 const routes: Routes = [
   {
     path: '',
-    loadChildren: () => import('./pages/home/home.module').then((m) => m.HomeModule),
```

Comment:
out of curiosity can you share me insight or links why angular does not need this anymore 😁

Reply:
This is a manual change I have done as ng 15 supports this now.. If you `export export HomeModule` you can just specify the filelocation.

Reply:
got it, thanks